src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java

```diff
@@ -649,10 +651,99 @@ private boolean isValidUserDirection(
                     .orElse(false);
 
                 return callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
-                       && callback.getEvent() == Event.FTPA_DECISION_LEADERSHIP_JUDGE
+                       && (callback.getEvent() == Event.LEADERSHIP_JUDGE_FTPA_DECISION || callback.getEvent() == Event.RESIDENT_JUDGE_FTPA_DECISION)
                        && isPositive;
```

Comment:
Similar to above. I would be in favour of a variable named 'isPostiveOutcome' instead.